doc/architecture/decisions/0013-refer-to-lost-beds-as-out-of-service-beds-in-future-features.md

```diff
@@ -0,0 +1,25 @@
+# 13. Refer to 'lost beds' as 'out of service beds' in future features
+
+Date: 2024-05-16
+
+## Status
+
+Tentative
+
+## Context
+
+- We will soon iterate on the 'Manage' transaction of the service.
+- The North East (NE) AP area are currently the only area using this transaction.
+- We need to retain 'Manage' features and ensure their stability whilst rapidly developing overlapping functionality.
+- The most critical feature is currently referred to as 'lost beds'.
+- Users are more familiar with the term 'out of service beds'.
+
+## Decision
+
+We will call the features intended to replace the 'lost beds' functionality 'out of service beds'.
+
+## Consequences
+
+- We will be able to build the new functionality without the changes polluting the existing code enabling us to iterate quickly.
+- We will have similar functionality under different names - it may not be clear to incoming developers which is the new and which is functionality being replaced
```

Comment:
If I understand your point here, would it be clearer to put this in a positive form e.g.

> The different terminology ("lost beds" versus "out-of-service beds") should help incoming developers differentiate between the new code and the functionality which is being replaced